cmake/dependencies.cmake

```diff
@@ -345,18 +345,19 @@ if(ENABLE_INTEL_GNA)
 	    SHA256 ${GNA_HASH})
         update_deps_cache(GNA_EXT_DIR "${GNA_EXT_DIR}" "Path to GNA root folder")
         debug_message(STATUS "gna=" ${GNA_EXT_DIR})
+        if (WIN32)
+            set(GNA_PLATFORM_DIR win64 CACHE STRING "" FORCE)
+        elseif (UNIX)
+            set(GNA_PLATFORM_DIR linux CACHE STRING "" FORCE)
+        else ()
+            message(FATAL_ERROR "GNA not supported on this platform, only linux, and windows")
+        endif ()
+        set(GNA_LIB_DIR x64 CACHE STRING "" FORCE)
+        set(GNA_PATH ${GNA_EXT_DIR}/${GNA_PLATFORM_DIR}/${GNA_LIB_DIR} CACHE STRING "" FORCE)
+    else()
+        find_library(GNA_PATH gna)
```

Comment:
If GNA lib is not found , show error message and stop further build.. message(FATAL_ERROR

Reply:
Updated for both.